R/runSearch.R

```diff
@@ -137,6 +138,20 @@ runSearch <- function(keywords,
       dplyr::rename_with(tolower)
   }
 
+  # collect the drug_strength table if drug
+  if (domains == "drug"){
```

Comment:
should this be %in% ? The user can search for codes in multiple domains at the same time. What do we want the behaviour to be if the user searches both conditions and drugs at the same time?

Reply:
https://github.com/darwin-eu/CodelistGenerator/blob/3598abc543c83647963ae4ac304141c28ba22d27/R/getCandidateCodes.R#L171-L179

But with these, to my understand, the `domains` goes into the `runSearch` function will always be one?  unless the user run the `runSearch` function directly?   But I can change it to `"drug" %in% tolower(domains) `

Reply:
Ah yes, you´re right!